src/Microsoft.Extensions.Logging.Abstractions/ILoggerFactory.cs

```diff
@@ -19,9 +19,13 @@ public interface ILoggerFactory : IDisposable
         ILogger CreateLogger(string categoryName);
 
         /// <summary>
+        /// <para>
+        /// This method is obsolete, use AddProvider on the concrete Microsoft.Extensions.Logging.LoggerFactory class.
```

Comment:
`This method is obsolete and will be removed in a future version. The recommended alternative is to call the Microsoft.Extensions.Logging.AddConsole() extension method on the Microsoft.Extensions.Logging.LoggerFactory instance.`